### PR TITLE
fix(scripts/ZulGurub): Update Gahzranka's abilities

### LINKED ISSUE #11562: [ZG] Gahz'ranka - Missing cast time
### Current Behaviour

Frost Breath (16099) and Massive Geyser (22421) are missing its 2s and 1.5s cast times, respectively.

When used by players, both spells have normal cast times.

### Expected Blizzlike Behaviour

Gahz'ranka should stop and cast both spells to their full durations.

### Source

https://youtu.be/JEmFTdr65UY?t=88

### Steps to reproduce the problem

1. `.additem 19975 5`
2. `.additem 19974`
3. `.go xyz -11695 -1770 13 309` Teleports to Pagle's Pointe
4. Engage Gahz'ranka in combat and observe the lack of casting time

### Extra Notes

Edit: From looking at a couple of videos I'd say that the timers are around 10-30s for Frost Breath and 12-30s for Massive Geyser

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/59e45c251eb53abb0ed4b37a57254cbdf415e69a

### Operating system

Windows 10

### Custom changes or Modules

_No response_

### LINKED ISSUE #11568: [ZG] Gahz'ranka - Thrash spell missing.
### Current Behaviour

Gahz'ranka does not thrash (ID: 3391)

### Expected Blizzlike Behaviour

Gahz'ranka should occasionally thrash

### Source

>Gahz'ranka occasionally thrashes, gaining 2 extra attacks on his next swing. ([source](https://wowpedia.fandom.com/wiki/Gahz'ranka))

Although the ability was only added to the article during WoW Classic, there are also videos during WotLK that showcase Gahz'ranka making several attacks at once.

https://youtu.be/JEmFTdr65UY?t=180, 2:01, 2:12 at 4:03, etc

### Steps to reproduce the problem

1. `.additem 19975 5`
2. `.additem 19974`
3. `.go xyz -11695 -1770 13 309` Teleports to Pagle's Pointe
4. Engage Gahz'ranka

### Extra Notes

[Thrash](https://wowgaming.altervista.org/aowow/?spell=3391) (ID: 3391)

Edit: From looking at a couple of videos, it looks like the timer for Thrash is the most random. I'd say around 9-20s

### AC rev. hash/commit

https://github.com/azerothcore/azerothcore-wotlk/commit/59e45c251eb53abb0ed4b37a57254cbdf415e69a

### Operating system

Windows 10

### Custom changes or Modules

_No response_

### PR DESCRIPTION
**Pull request by Si1ker**
Pull Request by [sílker](https://www.chromiecraft.com/en/armory/?character/ChromieCraft/s%C3%ADlker)

## Changes Proposed:
- Disable triggered spells
- Adjust init spell events
- Adjust CD timers
- Add thrash spell/event

## Issues Addressed:

- Closes #11562
- Closes #11568

## SOURCE:

https://www.youtube.com/watch?v=JEmFTdr65UY
The encounter starts at 1:00

## Tests Performed:

- None

## How to Test the Changes:

1. Enter ZUl'Gurub
2. Engage the boss
3. Check spell casting

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/EasternKingdoms/ZulGurub/boss_gahzranka.cpp [hunk 1]
```diff
@@ -82,12 +82,12 @@ class boss_gahzranka : public CreatureScript // gahzranka
                 switch (eventId)
                 {
                     case EVENT_FROSTBREATH:
-                        DoCastVictim(SPELL_FROSTBREATH, true);
-                        events.ScheduleEvent(EVENT_FROSTBREATH, urand(7000, 11000));
+                        DoCastVictim(SPELL_FROSTBREATH, false);
```
Comment (Nyeriah) [inline src/server/scripts/EasternKingdoms/ZulGurub/boss_gahzranka.cpp:85]: it's false by default, the parameter can be supressed

Comment (Si1ker) [reply src/server/scripts/EasternKingdoms/ZulGurub/boss_gahzranka.cpp:85]: Yep, I just checked the ac dir for some examples. Back then we used to have it even on default non-triggered spells.
insert garrosh meme: times changes.jpg

**Comment by Si1ker** [comment]
@Nyeriah this is ready to go.

**Comment by Nyeriah** [comment]
Where are the timers taken from? the video?

**Comment by Nyeriah** [comment]
If someone could crossreference this with other sources that would be great 👍

**Comment by Gultask** [comment]
Just found out that [TC-3.3.5 has timers for Gahz'ranka](https://github.com/TrinityCore/TrinityCore/blob/3.3.5/src/server/scripts/EasternKingdoms/ZulGurub/boss_gahzranka.cpp). Frost Breath seems far too frequent, though.
My report has some timers but the conclusions I made were not very good. Here's what I've gathered from WotLK videos only:

Frost Breath: [10, 11, 13, 13, 14, 16, 18, 19, 20, 20, 20, 21, 25, 28, 54] avg: 20
Slam: [19, 20, 23, 24, 25, 25, 27, 27, 28, 30, 32, 36, 56] avg: 28,6
Geyser: [13, 14, 15, 15, 18, 19, 21, 22, 23, 23, 24, 25, 26, 27, 29] avg: 21
Thrash: [9, 9, 10, 13, 14, 17, 18, 20, 20, 26, 32] avg: 17

I could go back and check for more WotLK videos to be more precise if needed, I spent way more time on Mandokir than on Gahz'ranka.

**Comment by Gultask** [comment]
[Vmangos has 8-20s to the Frost Breath timer](https://github.com/vmangos/core/blob/development/src/scripts/eastern_kingdoms/stranglethorn_vale/zulgurub/boss_gahzranka.cpp), which looks much better than 7-11s

**Comment by Si1ker** [comment]
@Nyeriah merge.

**Pull request merged**